packages/listbox/test-suites/ListboxMixin.suite.js

```diff
@@ -96,24 +96,6 @@ export function runListboxMixinSuite(customConfig = {}) {
         );
       });
 
-      it('throws if a child element with a different name than the group tries to register', async () => {
```

Comment:
Duplicate test, implementation is not in ListboxMixin but rather in ChoiceGroupMixin

Reply:
you now longer throw at all?

Reply:
Not in this instance no, it just syncs it to the parent's name without error'ing. If I kept the error, the situation would be this:

- parent name updates --> child name !== parent name --> error. So we have to add an edge case to not error in this case
- child formElement gets added --> child name !== parent name --> still error.

I figured it was better to align behaviours and just sync in all cases. I don't like the error, it's a bit unhelpful, and in all cases the solution to fix the error is to sync the name with the parent first. So why not do this by default for our users and prevent them from having to do manual work.